platform/core/src/services/HangingProtocolService/HangingProtocolService.ts

```diff
@@ -135,6 +140,7 @@ export default class HangingProtocolService extends PubSubService {
     this._commandsManager = commandsManager;
     this._servicesManager = servicesManager;
     this.protocols = new Map();
+    this.protocolsCopy = new Map();
```

Comment:
Please initialize in the declaration object - makes it immediately obvious what it is.